help_strings.json

```diff
@@ -0,0 +1,51 @@
+{
+    "rules": [
+      "Type '!random' in chat and the bot will play ",
```

Comment:
Is there a reason this is an array of split clauses?

Reply:
readability

Reply:
I think it'd make more logical sense to keep it as a single string. Text wrapping should be handled by the IDE